front/pages/w/[wId]/builder/assistants/index.tsx

```diff
@@ -331,6 +340,13 @@ export default function WorkspaceAssistants({
         hasTopPadding={false}
         isWideMode
       >
+        <DeleteAssistantsDialog
```

Comment:
Wouldn't it be nicer to have one Dialog for both bulk edit and single edit? I can make a followup PR and adjust a few UI elements too if you are okay with it 😄

Reply:
Yes I thought about it, can be done in another PR if you want :-)